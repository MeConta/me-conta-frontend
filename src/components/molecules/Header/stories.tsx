import { Story, Meta } from '@storybook/react'

import { Header } from '.'

export default {
  component: Header,
  title: 'Molecules/Header',
  argTypes: {
    unoptimizedImage: {
      control: false
    }
  }
} as Meta

export const Default: Story = () => <Header unoptimizedImage />
