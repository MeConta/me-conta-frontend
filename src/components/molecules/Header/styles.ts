import styled from 'styled-components'

export const Header = styled.header`
  color: red;
`
