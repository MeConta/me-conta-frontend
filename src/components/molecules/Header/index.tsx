import React from 'react'
import * as S from './styles'

export default function Header() {
  return <S.Header>Header</S.Header>
}
