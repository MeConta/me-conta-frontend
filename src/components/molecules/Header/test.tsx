import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Header from './index'

it('should render Header', () => {
  const { getByText } = render(<Header />)
  expect(getByText('Header')).toBeInTheDocument()
})

test('it works', () => {
  const tree = renderer.create(<Header />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('color', 'red')
})
