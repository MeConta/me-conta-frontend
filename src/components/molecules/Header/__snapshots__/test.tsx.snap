// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  color: red;
}

<header
  className="c0"
>
  Header
</header>
`;
